docs/fundamentals/diagnostics/sample/src/DiagDemo/project.json

```diff
@@ -0,0 +1,34 @@
+{
+    "webroot": "wwwroot",
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "Microsoft.AspNet.Server.IIS": "1.0.0-beta4",
+        "Microsoft.AspNet.Server.WebListener": "1.0.0-beta4",
+        "Microsoft.AspNet.Diagnostics": "1.0.0-beta4",
+        "Kestrel": "1.0.0-beta4"
+    },
+
+    "commands": {
+        "web": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.WebListener --server.urls http://localhost:5000",
+        "kestrel": "Microsoft.AspNet.Hosting --server Kestrel --server.urls http://localhost:12344"
+    },
+
+    "frameworks": {
+        "dnx451": { }/*,
```

Comment:
`/*...*/` comments are no longer supported, remove them.  The sample should work on dnxcore50.